sandbox/write-trimmomatic.py

```diff
@@ -16,12 +17,12 @@
     print """\
 mkdir trim
 cd trim
-java -jar /usr/local/bin/trimmomatic-0.30.jar PE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:/usr/local/share/adapters/TruSeq3-PE.fa:2:30:10
-/usr/local/share/khmer/scripts/interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
+TrimmomaticPE ../%s ../%s s1_pe s1_se s2_pe s2_se ILLUMINACLIP:%s:2:30:10
+interleave-reads.py s1_pe s2_pe | gzip -9c > ../%s
 
-cat s1_se s2_se | gzip -9c > ../%s
+cat s1_se s2_se | gzip -9c > ../../trimmed/%s
 cd ..
 rm -r ./trim/
 
 chmod u-w %s %s
-""" % (r1, r2, final_pe, final_se, final_pe, final_se)
+""" % (r1, r2, sys.argv[1], final_pe, final_se, final_pe, final_se)
```

Comment:
@mr-c  Why we need this "sys.argv[1]"? I think also function doesn't take arguments?